src/main/webapp/search-style.css

```diff
@@ -0,0 +1,42 @@
+.hidden {
+  display: none;
+}
+
+label, select {
+  margin-bottom: 20px;
+}
+
+#login-box {
+    margin:auto;
+    width: 120px;
+    padding: 30px;
```

Comment:
Weird extra spacing here. Is prettier doing this?

Reply:
I think so - I'll go through and try to fix it. Prettier is being pretty weird.